packages/tessellate-request/lib/configuration/oauth-configuration-file-loader.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import nconf from 'nconf'
+import fs from 'fs'
+import type {OAuthConfig} from './oauth-configuration'
+
+let configurations: Array<OAuthConfig>
+
+export function init() {
```

Comment:
Added lazy loading and the code is now located in file `file-configuration-loader`. 

I did not add an `id` parameter to the method, because this class is only responsible for loading the configurations from a file. To actually retrieve a specific configuration (e.g. by an id), `configuration-provider` should be used.